rulebook.tex

```diff
@@ -2486,10 +2517,11 @@ \subsection{Changes compared to the Main Competition}
 \end{table}
 
 \paragraph{Orders}
-Unless specified otherwise, orders that have to be fulfilled in challenges
-are entered through the web shop
-(see the \ac{refbox} workshop in \refsec{sec:qualification})
-by any member of the team.
+Orders that have to be fulfilled in challenges can be chosen by the team
+attempting them by either entering them through
```

Comment:
Suggestion: Orders to be fulfilled in challenges can be chosen by the competing team by either ordering them through the webshop  or by configuring them in the refbox directly.